google/cloud/bigquery/query.py

```diff
@@ -1310,6 +1310,50 @@ def num_dml_affected_rows(self):
         if num_dml_affected_rows is not None:
             return int(num_dml_affected_rows)
 
+    @property
+    def created(self) -> Optional[datetime.datetime]:
+        """Datetime at which the job was created.
+
+        See:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobStatistics.FIELDS.creation_time
```

Comment:
I don't think this is available on the Query response object, is it?

Reply:
Please update the link to the correct object: http://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#body.QueryResponse.FIELDS.creation_time